Author: Petter Reinholdtsen <pere@hungry.com>
Description:
 Make sure the code compiles when using -ansi. Renames non-ANSI C 'inline' to
 '__inline'.

Index: trunk/types.h
===================================================================
--- trunk.orig/types.h
+++ trunk/types.h
@@ -31,7 +31,7 @@ typedef struct {
 #endif
 
 #if defined(ALIGNMENT_WORKAROUND) || defined(BIGENDIAN)
-static inline u64 U64(u32 low, u32 high)
+static __inline u64 U64(u32 low, u32 high)
 {
 	u64 self;
 
